setup.py

```diff
@@ -15,19 +15,20 @@ def read(fname):
     keywords=["prometheus", "openstack", "exporter"],
     url="https://github.com/CanonicalLtd/prometheus-openstack-exporter",
     scripts=["prometheus-openstack-exporter"],
-    install_requires=["prometheus_client",
-                      "python-keystoneclient<=3.10.0",
-                      "python-novaclient==6.0.0",
-                      "python-neutronclient<=6.1.0",
-                      "python-cinderclient",
-                      "netaddr",
-                      "swift"
+    install_requires=[
+        "prometheus_client",
+        "python-keystoneclient<=3.10.0",
+        "python-novaclient==6.0.0",
+        "python-neutronclient<=6.1.0",
+        "python-cinderclient",
+        "netaddr",
+        "swift"
```

Comment:
Best to include a trailing "," here so we don't have to change this line if appending something to this list in the future.

Reply:
Fixed.